pillow_heif/options.py

```diff
@@ -48,3 +48,12 @@
 To learn more read: `here <https://github.com/strukturag/libheif/issues/784>`_
 
 When use pillow_heif as a plugin you can set it with: `register_*_opener(allow_incorrect_headers=True)`"""
+
+
+SAVE_NCLX_PROFILE = False
+"""Should be `nclx` profile saved or not.
+
+.. note:: macOS viewer ``does not support opening`` images with **nclx** profiles, so this is disabled by default.
```

Comment:
Hi. Did you test this? The HEIC file I was commenting about in #118 does show correctly in the macOS "Preview" app and in Finder "quick look" feature, even though it has nclx information set on two image streams. Where can I find the documentation of the `macOS_compatibility_workaround_no_nclx_profile` flag?

Reply:
https://github.com/strukturag/libheif/blob/bcb2497474630263cf481539b25088679ae647e2/libheif/context.cc#L2520-L2523

https://github.com/strukturag/libheif/blob/bcb2497474630263cf481539b25088679ae647e2/libheif/heif.cc#L2841-L2856


Unfortunately, documentation is not the strong side of `libheif`.
And yeah, I tested today on last macOS(13.4) - Viewer was able to open a test photo.
Need to think what can be putted into description, probably better to ask in `libheif` repo

Reply:
I asked about it here: https://github.com/strukturag/libheif/discussions/931

Reply:
@wiggin15 https://github.com/strukturag/libheif/discussions/931#discussioncomment-6520340